sdk/communication/AzureCommunicationChat/Source/Signaling/CommunicationSignalingClient.swift

```diff
@@ -36,11 +48,11 @@ class CommunicationSignalingClient {
     private var communicationHandlers: [ChatEventId: CommunicationHandler] = [:]
 
     init(
-        skypeTokenProvider: CommunicationSkypeTokenProvider,
+        communicationSkypeTokenProvider: CommunicationSkypeTokenProvider,
         logger: ClientLogger = ClientLoggers.default(tag: "AzureCommunicationSignalingClient")
     ) throws {
```

Comment:
CommunicationSignalingClient is internal, yes? Otherwise this is a breaking change.

Reply:
yes its internal